### PR TITLE
Add info on how to include in projects

### PR DESCRIPTION
**Pull request by Maxr1998**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,3 +122,21 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
 	}
 });
 ```
+
+## Include into project
```
Comment (nielsvanvelzen) [inline README.md:126]:
```suggestion
## Install
```

File: README.md [hunk 2]
```diff
@@ -122,3 +122,21 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
...
+
+The Jellyfin api client is available through [Bintray/JCenter](https://bintray.com/jellyfin/jellyfin-apiclient-java/jellyfin-apiclient-java), and can be included in projects like any other dependency:
```
Comment (nielsvanvelzen) [inline README.md:128]:
```suggestion
The apiclient is available through [JCenter](https://bintray.com/jellyfin/jellyfin-apiclient-java/jellyfin-apiclient-java), and thus can be installed via Gradle like any other dependency:
```

File: README.md [hunk 3]
```diff
@@ -122,3 +122,21 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
...
+// build.gradle
+dependencies {
+	implementation 'org.jellyfin.apiclient:android:0.7.2'
+}
+```
+
+```kotlin
+// build.gradle.kts
+dependencies {
+	implementation("org.jellyfin.apiclient:android:0.7.2")
```
Comment (nielsvanvelzen) [inline README.md:140]:
```suggestion
	val apiclientVersion = "0.7.2"
	
	// For non Android projects
	implementation("org.jellyfin.apiclient:library:$apiclientVersion")
	
	// For Android apps (automatically includes the library and models)
	implementation("org.jellyfin.apiclient:android:$apiclientVersion")
```

File: README.md [hunk 4]
```diff
@@ -122,3 +122,21 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
...
 ```
+
+## Include into project
+
+The Jellyfin api client is available through [Bintray/JCenter](https://bintray.com/jellyfin/jellyfin-apiclient-java/jellyfin-apiclient-java), and can be included in projects like any other dependency:
+
+```groovy
+// build.gradle
+dependencies {
+	implementation 'org.jellyfin.apiclient:android:0.7.2'
```
Comment (nielsvanvelzen) [inline README.md:133]: Move the Groovy sample down (Kotlin-first) and rewrite with my suggestion

File: README.md [hunk 5]
```diff
@@ -122,3 +122,21 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
...
+}
+```
+
+```kotlin
+// build.gradle.kts
+dependencies {
+	implementation("org.jellyfin.apiclient:android:0.7.2")
+}
+```
```
Comment (nielsvanvelzen) [inline README.md:142]: Move this section up above the code samples

File: README.md [hunk 6]
```diff
@@ -122,3 +122,21 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
...
+The Jellyfin api client is available through [Bintray/JCenter](https://bintray.com/jellyfin/jellyfin-apiclient-java/jellyfin-apiclient-java), and can be included in projects like any other dependency:
+
+```groovy
+// build.gradle
+dependencies {
+	implementation 'org.jellyfin.apiclient:android:0.7.2'
+}
+```
+
+```kotlin
```
Comment (nielsvanvelzen) [inline README.md:137]: add `repositories { jcenter() }`

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Maxr1998) [reply README.md:126]: Don't know if "Install" is the right word, maybe "Setup"?

Comment (nielsvanvelzen) [reply README.md]: Setup sounds good!

**Review by nielsvanvelzen — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -42,6 +42,44 @@
...
+The apiclient is available through [JCenter](https://bintray.com/jellyfin/jellyfin-apiclient-java/jellyfin-apiclient-java), and thus can be installed via Gradle like any other dependency:
+
+```kotlin
+// build.gradle.kts
+repositories {
+	jcenter()
+}
+
+dependencies {
+	val apiclientVersion = "0.7.2"
```
Comment (nielsvanvelzen) [inline README.md:56]:
```suggestion
```
I think we shouldn't actually mention the version in this snippet so users won't end up using an older version if we don't update the README. The actual version to use is shown in the releases page and the badge on top of the readme.

Comment (Maxr1998) [reply README.md:56]: Right, updating the README is always a bit of a problem. Do we still keep the version variable, just with `…` in the string? I think that'd be the most obvious.

Comment (nielsvanvelzen) [reply README.md]: yes

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**